rest-twitter/TweetRetriever.py

```diff
@@ -0,0 +1,62 @@
+import os
+from birdy.twitter import AppClient, TwitterClientError
+
+class TweetRetriever:
+
+    def __init__(self):
+        '''Initialize TweetRetriever. For security, API keys associated with
+            your Twitter app should be present in environment variables.'''
+
+        # Load consumer keys from environment variables.
+        self.consumer_key = os.environ.get('TWITTER_API_KEY')
+        self.consumer_secret = os.environ.get('TWITTER_API_KEY_SECRET')
+        # Ensure API keys are present in environment variables.
+        if None in [self.consumer_key, self.consumer_secret]:
+            raise EnvironmentError('At least one Twitter API key is not present in the environment')
+        else:
+            print('Twitter API key and secret key loaded')
+        self.client = self._initialize_client()
+        if self.client is not None:
+            self.resource_search_tweets = self.client.api.search.tweets
+        else:
+            print('Failed to initialize TweetRetriever')
+
+    def _initialize_client(self):
+        '''Initialize the birdy API client'''
+
+        try:
+            # Initialize our client object with consumer keys.
+            client = AppClient(self.consumer_key, self.consumer_secret)
+            # Obtain the OAuth2 access token using our consumer keys.
+            self.access_token = client.get_access_token()
+            # Re-initialize our client object that stores the access token for later use.
+            client = AppClient(self.consumer_key, self.consumer_secret, self.access_token)
+        except TwitterClientError as ex:
+            print('Connection or access token retrievel error:' + str(ex))
+            client = None
+        return client
+
+    def get_tweets(self, trend, latitude, longitude, radius='25mi'):
+        '''Returns tweets from Twitter's search/trends endpoint; there may be zero tweets!'''
+
+        # Enclose trend in double-quotes for exact matching unless it's a hashtag
+        trend = '"' + trend + '"' if trend[0] == '#' else trend
```

Comment:
can we add a log here?

Reply:
yes, let's add it as a feature in a future pull request